src/app.controller.ts

```diff
@@ -7,6 +7,8 @@ export class AppController {
 
   @Get()
   getHello(): string {
+    console.log('hi');;
+    console.log('hello');
```

Comment:
console.log('cap-team');;;;
```suggestion
    console.log('hello');
    console.log('cap-team');;;;
    console.log('cap-team');;;;
```